derex/runner/mongodb.py

```diff
@@ -44,6 +55,27 @@ def inner(*args, **kwargs):
     return inner
 
 
+def execute_root_shell(command: Optional[str]):
+    """Open a root shell on the MongoDB database. If a command is given
+    it is executed."""
+    args = [
+        "run",
```

Comment:
This will work only if the mysql container is already running. We can speed things up by not creating a new container, using `exec` instead of `run`